src/tag/Handler.hxx

```diff
@@ -75,8 +77,11 @@ tag_handler_invoke_tag(const TagHandler &handler, void *ctx,
 	assert((unsigned)type < TAG_NUM_OF_ITEM_TYPES);
 	assert(value != nullptr);
 
-	if (handler.tag != nullptr)
-		handler.tag(type, value, ctx);
+	if (handler.tag != nullptr) {
+		Iconv::Converter conv;
+		std::string v = conv.toUTF8(value, strlen(value));
```

Comment:
This doesn't belong here

Reply:
Just for caryaudio. Convert some chinese chaset, which may GB2312, BIG5 or GBK coded, to UTF-8.